src/base/preferences.h

```diff
@@ -313,6 +313,8 @@ class Preferences : public QObject
     void setCloseToTrayNotified(bool b);
     TrayIcon::Style trayIconStyle() const;
     void setTrayIconStyle(TrayIcon::Style style);
+    bool iconsInMenusEnabled() const;
+    void setIconsInMenusEnabled(bool enable = true);
```

Comment:
The function should not provide a default value.

Reply:
I see. Fixed.